### PR TITLE
feat: added Kube-green module README

### PR DESCRIPTION
**Pull request by emanuele232**
## What this PR is for?

This PR adds the README for the module scheduling/kube-green
It describes the structure of the module , its flavours and how to use it.

### CODE REVIEW

File: modules/scheduling/kube-green/base/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Traefik Module
+
+Kube-green is a Kubernetes addon that allows you to shut down the workload you don't need with the definition of *schedules*.
```
Comment (JGiola) [inline modules/scheduling/kube-green/base/README.md]:
```suggestion
Kube-green is a Kubernetes addon that allows you to shut down the workloads you don't need with the definition of *schedules*.
```

File: modules/scheduling/kube-green/base/README.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+[Here is the link to Kube-green official repository.]
```
Comment (JGiola) [inline modules/scheduling/kube-green/base/README.md:5]: I think that here a link is missing 😄

File: modules/scheduling/kube-green/base/README.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+## What the base module contains
+
+- **CRDs:** Kube-green custom resource definitions
+- **Resources (Traefik ingress):**
```
Comment (JGiola) [inline modules/scheduling/kube-green/base/README.md]:
```suggestion
- **Resources:**
```

typo from copy and paste

File: modules/scheduling/kube-green/base/README.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+  - **Configs:** configurations of related resources.
+  - **Workload:** deployments and services for Kube-green ingress and its dashboards
```
Comment (JGiola) [inline modules/scheduling/kube-green/base/README.md]:
```suggestion
  - **Workload:** deployments and services for Kube-green controller and its service
```

File: modules/scheduling/kube-green/base/README.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+## Flavors
+
+This module has no flavors
+
+## Compatibility Matrix
+
+| Module Version | Tool Version   |
+|----------------|----------------|
+| 1.24.0         | 0.5.0          |
```
Comment (JGiola) [inline modules/scheduling/kube-green/base/README.md]:
```suggestion
| 1.24.0         | 0.4.0          |
```

We haven't already update to 0.5.0

File: modules/scheduling/kube-green/base/README.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+Kube-green is a Kubernetes addon that allows you to shut down the workload you don't need with the definition of *schedules*.
+
+[Here is the link to Kube-green official repository.]
+
+## What the base module contains
+
+- **CRDs:** Kube-green custom resource definitions
+- **Resources (Traefik ingress):**
+  - **Configs:** configurations of related resources.
```
Comment (JGiola) [inline modules/scheduling/kube-green/base/README.md]: Better be explicit that we deploy a validation webhook for the kube-green resources

**Review by JGiola — CHANGES_REQUESTED** [review]

File: modules/scheduling/kube-green/base/README.md (hunk 2 above)
Comment (emanuele232) [reply modules/scheduling/kube-green/base/README.md:5]: added the doc link

**Review by JGiola — APPROVED** [review]

**Pull request merged**